src/test/java/br/com/sbk/sbking/core/boarddealer/Partial40CardDeckTest.java

```diff
@@ -0,0 +1,41 @@
+package br.com.sbk.sbking.core.boarddealer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import br.com.sbk.sbking.core.Card;
+import br.com.sbk.sbking.core.Rank;
+import br.com.sbk.sbking.core.Suit;
+
+public class Partial40CardDeckTest {
+
+  private Partial40CardDeck subject;
+
+  @Before
+  public void setup() {
+    this.subject = new Partial40CardDeck();
+  }
+
+  @Test
+  public void getDeckshouldReturnADeckWith40DifferentCards() {
+    Set<Card> set = new HashSet<Card>(subject.getDeck());
+    int TOTAL_NUMBER_OF_CARDS = 40;
+    assertEquals(TOTAL_NUMBER_OF_CARDS, set.size());
+  }
+
+  @Test
+  public void getDeckshouldReturnADeckWithAllRanksGreaterThanFour() {
+    Card fourOfHearts = new Card(Suit.HEARTS, Rank.FOUR);
+    for (Card card : subject.getDeck()) {
+      boolean isGreaterThanSix = card.compareRank(fourOfHearts) > 0;
```

Comment:
Shouldn't this be called `isGreaterThanFour` ?

Reply:
Ops, my bad :sweat_smile:

Reply:
Fixed. I have to be more careful with my copy and pastes :slightly_smiling_face:

Reply:
:laughing: